### DISCUSSION TITLE
Can some startup parameters be written to a py file

### DISCUSSION THREAD

**Discussion by zh199609** [seleniumbase/SeleniumBase#2700] [Q&A] [answered]
![image](https://github.com/seleniumbase/SeleniumBase/assets/32641509/73710d90-98ed-45ab-a229-ea625be88916)
The above parameters can be written to a py file,Does seleniumbase itself provide this feature

**Comment by mdmintz** [comment, marked as answer]
Add options to the addopts section of a pytest.ini file: https://github.com/seleniumbase/SeleniumBase/blob/e2c3a5e44285dd5b2bbc90c635d2517e72427ba3/examples/pytest.ini#L4
